src/renderers/shared/stack/reconciler/ReactUpdateQueue.js

```diff
@@ -246,8 +246,9 @@ var ReactUpdateQueue = {
     enqueueUpdate(internalInstance);
   },
 
-  enqueueElementInternal: function(internalInstance, newElement) {
-    internalInstance._pendingElement = newElement;
+  enqueueElementInternal: function(internalInstance, nextElement, nextContext) {
+    internalInstance._pendingElement = nextElement;
+    internalInstance._context = nextContext;
```

Comment:
Technically we should probably queue context here too (`_pendingContext` or something) but it looks like the difference is probably unobservable right now? So I guess this is fine. Can you add a TODO here though?